server/src-lib/Hasura/GraphQL/Validate/Types.hs

```diff
@@ -156,17 +161,23 @@ instance EquatableGType ObjTyInfo where
   getEqProps a = (,) (_otiName a) (Map.map getEqProps (_otiFields a))
```

Comment:
We'll also need to check if the interfaces that are implemented are same.

Reply:
Changed getEqProps to output a 3 part tuple, which includes HashSet of implementing interfaces